.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -50,6 +56,14 @@ If no open issue exists, you can open one here:
 https://github.com/uswds/uswds/issues/new/choose.
 -->
 
+## Related pull requests
+
+_Indicate if there are other pull requests necessary to complete this issue_
+
+<!--
+Some changes to the USWDS codebase require a change to the documentation site, and need a pull request in the uswds-site repo. This could include new or updated component documentation, new or updated settings documentation, or changelog entries. Add links to any related PRs in this section. If this change requires an update to the uswds-site repo, but that PR does yet exist, just make sure to note that here.
+-->
```

Comment:
These new additions are great!
One small thing: it would be good to introduce line breaks in these comments to make it more readable in GitHub source. One possibility:

```suggestion
<!-- 
Some changes to the USWDS codebase require a change to the documentation site, 
and need a pull request in the uswds-site repo.
This could include:
1. New or updated component documentation
2. New or updated settings documentation, or
3. Changelog entries. 
Add links to any related PRs in this section.
If this change requires an update to the uswds-site repo, but that PR does yet exist,
just make sure to note that here. 
-->
```

Reply:
I've added this change in 0d5d4513e48af35bb2dd0830c9de519bae91306c. I've also:

- Changed it to an unordered list to match previous lists
- Added additional new lines for clarity
- Moved text to new line after 80 characters
- Condensed last paragraph